pages/404.js

```diff
@@ -1,60 +1,22 @@
-import Link from 'next/link'
+import { ErrorPage } from '@dts-stn/service-canada-design-system'
 
-export default function Custom404() {
+export default function Custom404(props) {
   return (
-    <div className="grid md:grid-cols-2 sm:grid-cols-1 items-center justify-center overflow-visible md:h-96 sm:h-screen mx-2 my-2 px-20">
-      <div className="error-404">
-        <h1 className="text-2xl">{"We couldn't find that Web page"}</h1>
-        <h2>An error 404 occured on server</h2>
-        <p>
-          {
-            "We're sorry you ended up here. Sometimes a page gets moved or deleted, but hopefully we can help you find what you're looking for. What next?"
-          }
-        </p>
-        <ul>
-          <li>
-            Return to the{' '}
-            <Link href="/">
-              <a className="text-cyan-600 underline">home page</a>
-            </Link>
-            ;
-          </li>
-          <li>
-            <Link href="https://www.canada.ca/en/contact.html">
-              <a className="text-cyan-600 underline">Contact us</a>
-            </Link>
-            {" and we'll help you out."}
-          </li>
-        </ul>
-      </div>
-      <div className="error-404">
-        <h1 className="text-2xl">Nous ne pouvons trouver cette page Web</h1>
-        <h2>Erreur 404</h2>
-        <p>
-          {
-            "Nous sommes désolés que vous ayez abouti ici. Il arrive parfois qu'une page ait été déplacée ou supprimée. Heureusement, nous pouvons vous aider à trouver ce que vous cherchez. Que faire?"
-          }
-        </p>
-        <ul>
-          <li>
-            Retournez à la{' '}
-            <Link href="/">
-              <a className="text-cyan-600 underline">page {"d'accueil;"}</a>
-            </Link>
-          </li>
-          <li>
-            <Link href="https://www.canada.ca/en/contact.html">
-              <a className="text-cyan-600 underline">Communiquez avec nous</a>
-            </Link>
-            {" pour obtenir de l'aide."}
-          </li>
-        </ul>
-      </div>
-    </div>
+    <ErrorPage
+      lang={props.locale}
+      errType="404"
+      isAuth={!props.isAuth}
+      homePageLink={
+        props.locale === 'en' ? '/en/my-dashboard' : '/fr/mon-tableau-de-bord'
+      }
+      accountPageLink="/"
+    />
   )
 }
 
 export async function getStaticProps({ locale }) {
+  const langToggleLink = locale === 'en' ? '/fr/404' : '/404'
```

Comment:
Should be /en/404 for consistency

Reply:
It won't work since it will append to whats already there.

![image](https://user-images.githubusercontent.com/77116011/217060744-a1dd4994-f4be-4774-a4f8-7880162c9904.png)